src/main/java/de/flo56958/minetinker/modifiers/Modifier.java

```diff
@@ -200,7 +200,7 @@ protected void init(Material m) {
 
 		if (ConfigurationManager.getConfig("Modifiers.yml").getBoolean("UseCustomModelData", false)) {
 			this.modItem.setType(Material.STICK);
-			NBTUtils.getHandler().setInt(this.modItem, "CustomModelData", this.customModelData);
+			DataHandler.setInt(this.modItem, "CustomModelData", this.customModelData);
```

Comment:
CustomModelData is a vanilla value, so i think we still need something to write NBT-Data to the items directly. (could be over reflections)

Reply:
Or maybe we could use NamespaceKey.minecraft() for the "vanilla" NBT-Tags

Reply:
the minecraft namespace does NOT work, but there is a better method with ItemMeta. CMD is also a workaround for Optifine as the new Tags can not be read by Optifine sadly.